tests/specs/getLocalChanges.js

```diff
@@ -3,11 +3,11 @@ var dbFactory = require('../utils/db')
 
 /* create if db does not exist, ping if exists or created */
 test('api.getLocalChanges() creates new db', function (t) {
-  t.plan(1)
+  t.plan(2)
   var db = dbFactory()
   var PouchDB = db.constructor
   var remoteName = PouchDB.utils.uuid(10)
-  var api = db.hoodieLocalChanges({local: 'LCDB1',remote: 'LCDB2'})
+  var api = db.hoodieLocalChanges({local: 'LCDB1', remote: 'LCDB2'})
```

Comment:
there should be no `local` option, only `remote`. `local` is the `db` you call `.hoodieLocalChanges` on